### ISSUE TITLE
Blender 2.8 support

### ISSUE THREAD

**Issue by kevzettler** [chinedufn/blender-actions-to-json#5]
Making this issue to track work on supporting 2.8. Recently went to install Blender 2.8 and `blender-actions-to-json` on a new machine and seeing failures.

```
Kevs-MacBook-Pro:crashgiants kevzettler$ actions2json --install
Traceback (most recent call last):
  File "/Users/kevzettler/.config/yarn/global/node_modules/blender-actions-to-json/bin/install-addon.py", line 11, in <module>
    bpy.ops.wm.addon_install(filepath=addonFilePath)
  File "/Applications/Blender.app/Contents/Resources/2.80/scripts/modules/bpy/ops.py", line 201, in __call__
    ret = op_call(self.idname_py(), None, kw)
AttributeError: Calling operator "bpy.ops.wm.addon_install" error, could not be found
```

Heres a guide with some info on breaking changes and other update considerations
https://theduckcow.com/2019/update-addons-both-blender-28-and-27-support/

**Comment by chinedufn** [comment]
I recently converted a couple addons to 2.8 after installing it over the weekend and the conversion ended up being pretty easy

https://github.com/chinedufn/landon/commit/d7c694a2399369518c4f0c0a6ea99998e72397e9

https://github.com/chinedufn/blender-iks-to-fks/commit/71ac64fd6dc64b9d36985163d2133e4273d5a412

**Comment by chinedufn** [comment]
If you happen to get to it - my approach was just

```
npm run test
# Fix error
npm run test
```

Otherwise I can take a look at this!

**Comment by chinedufn** [comment]
I added Blender 2.80 support and published to npm as 3.0.0
